Software-Engineering/Overall-Ratings.md

```diff
@@ -7,7 +7,9 @@ We use Maturing, Performing, or Exceeding bands within each level to help us ens
 
 In short:
 - **Level** (i.e. L1-5) is used to set the person's role title (e.g. Junior Software Engineer, Senior Software Engineer), to broadcast expectations and reinforce archetypes, to the person, across Octopus and beyond.
-- **Band** (i.e. Maturing/Performing/Exceeding) is used to reflect value contributed within that level, and thereby help us pass the printer test.
+- **Band** (i.e. Maturing/Performing/Exceeding) is used to reflect value contributed within that level, and thereby help us pass the printer test. 
+
+**NB:** _The band is not a reflection of how well an individual is performing their job. It is possible to meet or exceed expectations at your performance review while operating in any of the three bands. Being promoted into the Maturing band typically indicates we have **increasing expectations** of someone._
```

Comment:
I'm not sure about this line in particular, but I wanted to address the feeling that some engineers have that continuing to be in the Maturing band is somehow equivalent to "Not Meeting Expectations" at performance review time.

Reply:
> continuing to be in the Maturing band is somehow equivalent to "Not Meeting Expectations" at performance review time

Ah, right.

💭 Assuming that we want to keep our two separate systems (i.e. 1. bands vs 2. [perf review overall ratings](https://docs.google.com/document/d/17illLT2-oGuHIuoHpzqlaSBbLj7muaSQBGwDXy1AQAM/edit)), then it might help to outline them both here explicitly, e.g:
> Our system for bands (Maturing/Performing/Exceeding) is _distinct from_ our system for "overall ratings" in performance reviews.
> 1. Bands (Maturing/Performing/Exceeding): [describe the purpose and meaning]
> 2. [Performance review overall rating (1-5)](https://docs.google.com/document/d/17illLT2-oGuHIuoHpzqlaSBbLj7muaSQBGwDXy1AQAM/edit): [describe the purpose and meaning]

Reply:
I've clarified that Level/Band describes what our expectations are, and that the Performance Review describes how well those expectations have been met.

I'd love to know if this resonates with you, @Waldo000000 ?

Reply:
> _The Band (Maturing/Performing/Exceeding) is distinct from the Performance Review Overall Rating (1-5). The combination of Level and Band describes the expectations that Octopus has of an engineer, while the Performance Review Overall Rating describes performance against those expectations._

Yup, I think that's an accurate and helpful description 👍